src/tarantool_schema.c

```diff
@@ -14,27 +14,41 @@
 #define MUR_SEED 13
 #include "third_party/msgpuck.h"
 
-int mh_indexcmp_eq(
-		const struct schema_index_value **lval,
-		const struct schema_index_value **rval,
-		void *arg
-) {
-	(void )arg;
-	if ((*lval)->key.id_len != (*rval)->key.id_len)
-		return 0;
-	return !memcmp((*lval)->key.id, (*rval)->key.id, (*rval)->key.id_len);
-}
+#define MH_DEFINE_CMPFUNC(NAME, TYPE)								\
```

Comment:
I trying to retain myself from formatting nitpicking, but here the entire block formatted with exceeding 80 symbols limit.